prerequisites.md

```diff
@@ -1,8 +1,9 @@
 # Prerequisites to the Tenant-in-a-day-training
 ## General information
 
-This guide is ensure the training proceeds as desired, with all participants having fullfilled all basic requirements. 
+This guide is ensure the training proceeds as desired, with all participants having fullfilled all basic requirements.
```

Comment:
Typo; "This guide is ensure" -> 'This guide aims to ensure'?
Suggestion; "requirements" -> 'pre-requisites'

Reply:
Fix'd.